src/ui/components/Comments/TranscriptComments/CommentCard.tsx

```diff
@@ -129,7 +129,7 @@ function CommentItem({
     showOptions = false;
   } else {
     content = (
-      <div className="space-y-4 text-xs break-words" style={{ lineHeight: "1.125rem" }}>
+      <div className="space-y-4 text-xs break-words whitespace-pre-wrap leading-comment-text">
```

Comment:
@jcmorrow `leading-5` didn't quite work since that's `1.25rem`. But I thought this was as good a place as any to start using more semantic class names (which will make @jonbell-lot23 exceedingly happy).

Reply:
I am exceedingly happy!